src/options.h

```diff
@@ -113,9 +113,9 @@
 #  define MEMDEBUG
 #  define KEEP_BODIES
 #  endif
-// Memory sanitizer also needs thread-local storage
+// Memory sanitizer also needs small memory model
```

Comment:
Also, of course the proper fix is to teach LLVM how to do TLS in the large memory model.